public/css/style.css

```diff
@@ -116,6 +119,61 @@ nav {
 
 #actorList {
   list-style: none;
+  max-height:30vh;
+  overflow: scroll;
```

Comment:
Excellent feedback on the scrollbars, thank you! I had no idea Macs hid them by default. No more scrollbars to be seen. Though the actor list does push the character assignment area down and even though it's showing the end of the list bleeding into that area, the delete buttons on those actor buttons aren't accessible. I figure since we will be splitting out soon, this isn't something to fret about yet.
https://user-images.githubusercontent.com/46881905/236970978-40faa205-ab8f-481a-a82f-2096ca2743fe.mp4